src/sync/background/index.ts

```diff
@@ -48,6 +54,40 @@ export default class SyncBackground extends SyncService {
             productVersion: options.appVersion,
         })
 
+        this.continuousSync = new MemexExtContinuousSync({
```

Comment:
The idea of the base `SyncService` is to prevent a lot of duplicate code setup. Maybe we could just pass in the `postReceive` processor to the parent constructor, since that's the only difference between the ext and app?

Reply:
Should be addressed in 583a05675e83150b9223e892fbd911b66bdd452d

Reply:
Forgot yesterday as I was finishing up to add the changes needed for this in the `memex-common` repo. Those are now in this PR:
https://github.com/WorldBrain/memex-common/pull/1

Should be a lot more simple of a diff to digest